inc/ruleimportcomputer.class.php

```diff
@@ -304,9 +304,7 @@ function findWithGlobalCriteria($input) {
 
       $it_criteria = [
          'SELECT' => 'glpi_computers.id',
-         'FROM'   => '', //to fill
          'WHERE'  => [], //to fill
-         'ORDER'  => 'glpi_computers.is_deleted ASC'
```

Comment:
Is that expected? As far as I know, OCS is the only one to use this piece of code currently.

Reply:
I was just wondering what this ORDER was for